paywall/next.config.js

```diff
@@ -10,6 +10,8 @@ module.exports = withSourceMaps({
     unlockEnv: process.env.UNLOCK_ENV || 'dev',
     httpProvider: process.env.HTTP_PROVIDER || '127.0.0.1',
     readOnlyProvider: process.env.READ_ONLY_PROVIDER,
+    paywallUrl: process.env.PAYWALL_URL,
+    paywallScriptUrl: process.env.PAYWALL_SCRIPT_PATH,
```

Comment:
`paywallScriptPath`?

Reply:
At least this would be better in terms of name since this is a path... and it looks like this is what you use below ;)

Reply:
Weird, I thought I did that :). Definitely intended to, maybe I didn't push the commit. In any case, will do!